app/src/main/java/net/gsantner/markor/format/csv/CsvConfig.java

```diff
@@ -0,0 +1,50 @@
+/*#######################################################
+ *
+ *   Maintained 2023 by k3b
+ *   License of this file: Apache 2.0
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.format.csv;
+
+import net.gsantner.opoc.format.GsTextUtils;
+
+/**
+ * Configuration for CSV file format.
+ *
+ * Implementation detail for csv support. This file should be not have dependencies to
+ * android and to Markor-Architecture.
+ */
+public class CsvConfig {
+    public static final CsvConfig DEFAULT = new CsvConfig(';', '"');
+    public static final char[] CSV_DELIMITER_CANDIDATES = {DEFAULT.getFieldDelimiterChar(), ',', '\t', ':', '|'};
+    public static final char[] CSV_QUOTE_CANDIDATES = {DEFAULT.getQuoteChar(), '\''};
+    private final char fieldDelimiterChar;
+    private final char quoteChar;
+
+    public CsvConfig(char fieldDelimiterChar, char quoteChar) {
+        this.fieldDelimiterChar = fieldDelimiterChar;
+        this.quoteChar = quoteChar;
```

Comment:
Use `m_` or `_` instead of `this.`. (Leaving this comment only once here, but applies to all new files)

Reply:
done